packages/api/portfolioDrafts/createPortfolioDraft.ts

```diff
@@ -23,7 +23,11 @@ export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayPr
     created_at: now,
     updated_at: now,
     status: ProvisioningStatus.NOT_STARTED,
+    name: "",
```

Comment:
Why are we setting an empty name here?

Reply:
You could just set the name attribute when doing the createPortfolioStep, by adding a new attribute in the UpdateExpression. I disagree with setting an empty name here.